packages/docs/docs/lambda/serverlesslambdasetup.md

```diff
@@ -0,0 +1,203 @@
+---
+image: /generated/articles-docs-lambda-serverlesslambdasetup.png
```

Comment:
- Title seems to generic for me - seems to suggest anybody should follow this setup guide, but I see it as only for people who cannot or don't want to use IAM. How about "Example setup without IAM"?
- Let's also regenerate the card like explained in the other comment

Reply:
I've resolved it by updating it to 'Example setup without IAM user'